django/db/migrations/autodetector.py

```diff
@@ -1033,6 +1035,10 @@ def _generate_added_field(self, app_label, model_name, field_name):
                     self.to_state,
                 )
             )
+        if depend_on_deletion:
+            dependencies.append(
+                (app_label, model_name, field_name, False),
+            )
```

Comment:
Do we need the `depend_on_deletion` flag? It seems we can always add the same field as a dependency because this is only the case only when we use `RemoveField`/`AddField` instead of `RenameField` :thinking: 
```suggestion
        dependencies.append((app_label, model_name, field_name, False))
```

Reply:
You're right, it's no problem to have the dependency even when we only add a field (from calling `generate_added_fields`) without a removal